### DISCUSSION TITLE
Topaz Extract Pre-trained Models

### DISCUSSION THREAD

**Discussion by joonp9** [tbepler/topaz#117] [Q&A]
Hello,

What are the differences among 4 pre-trained models in Topaz Extract (ResNet16 (32 and 64 units) and ResNet8 (32 and 64 units))? And is ResNet16 (64 units) the best out of the four as it is set as default (in cryoSPARC)?

Thank you,
Joon Park

**Comment by tbepler** [comment]
ResNet16 (64 units) is the best pre-trained model. Of the four models, the difference is size (number of parameters). ResNet16 (64) is the biggest and ResNet8 (32) is the smallest. When training new topaz models that are initialized from one the the pre-trained models, the consideration is amount of data, because the smaller models are less prone to overfitting but will not perform as well as the larger models if a lot of training data is available.

**Reply by joonp9** [reply]
Thank you so much for your explanation! One more thing. Is there recommended number of particles/micrographs for training?

**Reply by tbepler** [reply]
Generally somewhere in the 100-1000 particles range seems work well.

**Reply by joonp9** [reply]
Awesome! Thank you so much! Have a great day.
